device_samples/getting_started/src/connections/connection_string.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// Choose a protocol by uncommenting one of these transports.
+import { Mqtt as Protocol } from 'azure-iot-device-mqtt';
+// import { Amqp as Protocol } from 'azure-iot-device-amqp';
+// import { Http as Protocol } from 'azure-iot-device-Http';
+// import { MqttWs as Protocol } from 'azure-iot-device-mqtt';
+// import { AmqpWs as Protocol } from 'azure-iot-device-amqp';
+
+import { Client, Message } from 'azure-iot-device';
+
+// String containing Hostname, Device Id & Device Key in the following formats:
+//  "HostName=<iothub_host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>"
+const deviceConnectionString: string = process.env.IOTHUB_DEVICE_CONNECTION_STRING || '';
```

Comment:
nit: we could use the nullish coalescing operator (??) if we are using a reasonable new TypeScript compiler
```suggestion
const deviceConnectionString: string = process.env.IOTHUB_DEVICE_CONNECTION_STRING ?? '';
```

Reply:
Also, in some places it's `process.env.IOTHUB_DEVICE_CONNECTION_STRING || ''` and other places it's just `process.env.IOTHUB_DEVICE_CONNECTION_STRING`. Better to be consistent